tsconfig.json

```diff
@@ -2,13 +2,14 @@
   "compileOnSave": false,
   "compilerOptions": {
     "baseUrl": ".",
-    "declaration": true,
-    "emitDecoratorMetadata": true,
-    "experimentalDecorators": true,    
-    "skipLibCheck": true,
+    "incremental": false,
```

Comment:
Do you set incremental build to `false` because it is not working with babel for now?

Reply:
I did that because it failed in a certain scenario.. I think calling `yarn bootstrap --core` twice
It should not use a cache here because it removes dist and compiles everything from scratch

I just forgot to add that option for `yarn dev` in watch mode
Maybe we can do a tsconfig.prod.json and a tsconfig.json to add transparency in the tsc build process